i18n/messages-en.xtb

```diff
@@ -97,7 +97,7 @@
   <translation id="3386572067498503433" key="MSG_COMMON_COMPONENTS_CONDITIONS_CONDITIONSLIST_11" desc="Label when there is no data.">-</translation>
   <translation id="6464751813964841186" key="MSG_COMMON_COMPONENTS_CONDITIONS_CONDITIONSLIST_12" desc="Label when there is no data.">-</translation>
   <translation id="8142136741637546639" key="MSG_COMMON_COMPONENTS_CONDITIONS_CONDITIONSLIST_13" desc="Tooltip heartbeat column of pod conditions">Probed at </translation>
-  <translation id="6128317173209217017" key="MSG_COMMON_COMPONENTS_CONDITIONS_CONDITIONSLIST_14" desc="Tooltip transistion column of pod conditions">Probed at </translation>
+  <translation id="6128317173209217017" key="MSG_COMMON_COMPONENTS_CONDITIONS_CONDITIONSLIST_14" desc="Tooltip transition column of pod conditions">Probed at </translation>
```

Comment:
It has to be fixed directly in the template if it is English.

Reply:
Thanks for commenting, but this typo happens in the `desc` field not in the translation content field.

Reply:
All `xtb` files are autogenerated based on HTML template files. `desc` attributes are also taken from HTML templates. This  change would be overridden after running `gulp build:prod` anyway so it's not persistent.

Reply:
Got it, many thanks, the corresponding HTML template has been corrected before but the `xtb` files are not synced with it, what should I do now?

Reply:
It's a bug of the current system, that we use. You can edit them manually. In the new app version, we will work on new translation system.

Reply:
@maciaszczykm Does this PR need to be merged? If no needs to be merged, I will close this PR.

Reply:
At the moment it gives us no value. HTML is still not modified.

Reply:
Yes, I will close this issue.